Upgrade for Paints-UNDO

MackinationsAi opened the pull request:
Updated Gradio UI, Automatic File Saving, Metadata Image Generative Data Embedding, Streamlined Installation for Windows Users & Updated README.md for the new UI / features.

strawberrymelonpanda commented:
Just an opinion, but I don't feel the greeting adds much value here, and I prefer fewer prompts.

MackinationsAi commented:
Hey @strawberrymelonpanda, I'm totally fine with the greeting.json & corresponding naming function in the gradio_app.py script being removed.

MackinationsAi commented:
I'm also working on an additional feature that allows the user to input their own keyframes w/ corresponding operation steps. That way if an actual artist uses this build, they can input there own images/sketches as keyframes to give said user a bit more control & help guide the frame interpolation in the video generation more accurately.

MackinationsAi commented:
Removed the greetings.json & corresponding naming function in the gradio_app.py script. In addition, I added in a tab called Metadata Viewer that allows the user to drag & drop any of their initial_images key_frames or final_frames & the data will automatically appear in the DataViewer textbox. 

(I also updated the README.md by removing the section related to the greetings.json & added the Metadata Viewer tab to the Tutorials section)
